apps/meteor/client/views/room/Sidepanel/SidepanelItem/SidePanelParentDiscussion.tsx

```diff
@@ -0,0 +1,30 @@
+import type { SubscriptionWithRoom } from '@rocket.chat/ui-contexts';
+
+import SidePanelTag from './SidePanelTag';
+import SidePanelTagIcon from './SidePanelTagIcon';
+import { useParentDiscussionData } from './useParentDiscussionData';
+
+const SidePanelParentDiscussion = ({ room }: { room: SubscriptionWithRoom }) => {
+	const { prid } = room;
+
+	if (!prid) {
+		throw new Error('Parent room ID is missing');
+	}
```

Comment:
I think maybe we can come up with a way to avoid throwing these errors by being more specific with Typescript. Let's discuss this too.